game/src/sandbox/dashboards/commuter.rs

```diff
@@ -275,6 +276,33 @@ impl State for CommuterPatterns {
                     }
                 }
 
+                batch.push(Color::BLACK.alpha(0.5), block.shape.clone());
+
+                {
+                    // Indicate direction over current block
+                    let icon_name: &str;
+                    let icon_scale: f64;
+                    let from = self.composite.is_checked("from / to this block");
+                    if from {
+                        icon_name = "outward.svg";
+                        icon_scale = 1.2;
+                    } else {
+                        icon_name = "inward.svg";
+                        icon_scale = 1.0;
+                    };
+
+                    let center = block.shape.polylabel();
+                    let icon = GeomBatch::mapspace_svg(
```

Comment:
mapspace, not screenspace, right?

Reply:
Correct. That will just listen to `.scale(...)` to size it. `screenspace_svg` additionally scales by your monitor's DPI scaling factor. It's just for "screen-space" icons on UI panels, so people with Retina displays don't have super tiny text and icons. But anything in "map-space" shows up the same everywhere, because the player can pan/zoom. The only difference based on monitor size and DPI becomes how much stuff is shown on the map at a time.